### PR TITLE
i18n(es): Updated `middleware.mdx` & `integrations-reference.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
 Updated `middleware.mdx` & `integrations-reference.mdx` based on this commit e52561db526dccf3e08ae5fce9b685ce2168b923

### CODE REVIEW

File: src/content/docs/es/guides/middleware.mdx [hunk 1]
```diff
@@ -213,3 +213,14 @@ El valor de `locals` no se puede sobrescribir en tiempo de ejecución. Hacerlo p
...
+
+### `createContext`
+
+Una API de bajo nivel para crear un [`APIContext`](/es/reference/api-reference/#contexto-del-endpoint), que puede ser utilizado por el middleware de Astro.
+
+Esta función puede ser utilizada por las integraciones/adaptadores para ejecutar programáticamente el middleware de Astro.
+
+### `trySerializeLocals`
+
+Una API de bajo nivel que toma cualquier valor y trata de devolver una versión serializada (una string) de él. Si el valor no se puede serializar, la función lanzará un error en el runtime.
```
Comment (Waxer59) [inline src/content/docs/es/guides/middleware.mdx:226]:
```suggestion
Una API de bajo nivel que toma cualquier valor y trata de devolver una versión serializada (un string) de él. Si el valor no se puede serializar, la función lanzará un error en el runtime.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**